src/shared/utils/fetchApi.js

```diff
@@ -1,16 +1,16 @@
 import axios from "axios";
 
 export async function fetchApi({ url, method, body, propertyName }) {
-//   const user = JSON.parse(localStorage.getItem("user"));
-
-  const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI1NzM2MjMxNi0xZjNhLTRlY2QtYjA4Ni0xMzlmNWNmNjdhZjkiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.UQaoxva7UtUeVo47ndik81df4Gd_FfDSXVhNY9ehSq4"
+  const user = JSON.parse(localStorage.getItem("user"));
+ 
+//  const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI1NzM2MjMxNi0xZjNhLTRlY2QtYjA4Ni0xMzlmNWNmNjdhZjkiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.UQaoxva7UtUeVo47ndik81df4Gd_FfDSXVhNY9ehSq4"
 
   try {
     const { data, status} = await axios({
       method,
       url,
       data: body,
-      headers:{authorization:token}
+      headers: { authorization: user!== null ?user.token:"" },
```

Comment:
you can use && here if it works

Reply:
I have not used conditional rendering , it's nullish  coalescing operator .